.jshintrc

```diff
@@ -18,7 +18,7 @@
     "forin": false,
     "immed": true,
     "indent": 4,
-    "latedef": true,
+    "latedef": "nofunc",
```

Comment:
This will force jshint to ignore errors from undefined functions that are actually hoisted up.